tool/goinstall/installer_test.go

```diff
@@ -108,6 +110,7 @@ func TestInstaller_InstallTo(t *testing.T) {
 			i.goInstallRunner = tt.fields.goInstallRunner
 
 			got, err := i.InstallTo(tt.args.version, tt.args.destDir)
+			got = strings.ReplaceAll(got, string(os.PathSeparator), "/")
```

Comment:
This test fails for me because there is no `sh` by default on windows, regardless of the path. I wonder if we could do the same thing that the taskfile is doing and use https://github.com/mvdan/sh for these...